miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -268,8 +269,38 @@ internal class MiniAppDownloader(
         this.requireSignatureVerification = isRequired
     }
 
+    fun getCachedMiniApp(appId: String): Pair<String, MiniAppInfo> {
+        val miniAppInfo = miniAppStatus.getDownloadedMiniApp(appId)
+        return if (miniAppInfo != null) {
+            onGetCachedMiniApp(miniAppInfo)
+        } else {
+            throw MiniAppNotFoundException(MINIAPP_NOT_FOUND_OR_CORRUPTED)
+        }
+    }
+
+    fun getCachedMiniApp(appInfo: MiniAppInfo): Pair<String, MiniAppInfo> {
+        if (miniAppStatus.isVersionDownloaded(
+                appId = appInfo.id,
+                versionId = appInfo.version.versionId,
+                versionPath = storage.getMiniAppVersionPath(appInfo.id, appInfo.version.versionId)
+            )
+        ) {
+            return onGetCachedMiniApp(appInfo)
+        }
+        return throw MiniAppNotFoundException(MINIAPP_NOT_FOUND_OR_CORRUPTED)
```

Comment:
Since the `return` statement here is not reachable, we can remove it. However, we can also follow the similar style like `getCachedMiniApp(appId: String)` above.

e.g. 

```
return if (miniAppStatus.isVersionDownloaded( ... )) {
    onGetCachedMiniApp(appInfo)
}
else throw MiniAppNotFoundException(MINIAPP_NOT_FOUND_OR_CORRUPTED)
```